plugins/techdocs-module-addons-contrib/api-report.md

```diff
@@ -31,4 +31,7 @@ export type ReportIssueTemplateBuilder = ({
 
 // @public
 export const techdocsModuleAddonsContribPlugin: BackstagePlugin<{}, {}>;
+
+// @public
+export const TextSize: (props: unknown) => JSX.Element | null;
```

Comment:
Unknown type will be fixed here:
https://github.com/backstage/backstage/pull/11166/files#diff-03813941362a23884f6471cbbb50dd7cb7058cfe5cacabfff4e2612619610ec7